CHANGELOG.rst

```diff
@@ -13,6 +13,9 @@ Change Log
 Unreleased
 ~~~~~~~~~~
 
+* Add setting annotation linting.
+* Add feature toggle annotation linting.
```

Comment:
Wouldn't we need to bump version and release to use in edx-platform?

Reply:
:+1: I tagged v3.0.0